ui/lib/core/addon/components/info-table-row.hbs

```diff
@@ -29,7 +29,19 @@
         <Icon @name="minus" />
       {{/if}}
     </div>
-    <div class="column is-flex foobar" data-test-value-div={{@label}}>
+    <div class="column is-flex" data-test-value-div={{@label}}>
+      {{#if @addCopyButton}}
+        <div class="display-only">
```

Comment:
`display-only` class is from the `masked-input.scss` file and gives the button the appropriate styling (i.e., blue on hover, transparent background, smaller padding)